app/screens/EditLocationDetails.js

```diff
@@ -91,13 +92,14 @@ class EditLocationDetails extends Component {
 
         const operatorObj = operators.find(op=> op.id === operatorId) || {}
         const { name: operatorName = operator === -1 ? 'N/A' : 'Select operator' } = operatorObj
+        const keyboardDismissProp = Platform.OS === "ios" ? { keyboardDismissMode: "on-drag" } : { onScrollBeginDrag: Keyboard.dismiss }
```

Comment:
@RyanTG, after I merge this in, would you be ok if I clean this up a bit? Rather than passing it in as a prop to every Screen component, I'd rather put the logic in the screen component itself.

Reply:
Sure. I was only adding it to components that have Inputs under them, because those are the only times when keyboards are shown. So it's not for every Screen component - that's why I didn't include it in the component itself.

Reply:
Plus, it is also not only on Screen components. Just in case that's not clear.

Reply:
Yeah, I see it's being used all over. I'm just not a huge fan of doing this logic all over the app. I would really rather see us wrapping `ScrollView`, `FlatList`, etc with our own wrapper having this baked in. That being said, if this is working, we can cross that bridge sometime down the road.